esp/services/ws_workunits/ws_workunitsQuerySets.cpp

```diff
@@ -1176,13 +1176,13 @@ IPropertyTree* getQueriesOnCluster(const char *target, const char *queryset, Str
     try
     {
         StringBuffer control;
-        if (queryIDs.ordinality() == 0)
+        if (!queryIDs)
```

Comment:
I'd think !queryIDs or queryIDs->ordinality()==0 to make it easiest for callers.